src/frontend/staff-portal/src/app/components/ticket-info/ticket-info.component.ts

```diff
@@ -1,147 +1,575 @@
+      this.initialDisputeValues = this.setViolationTicketFieldsFromJSON(response);
+      this.lastUpdatedDispute = this.initialDisputeValues;
+      this.form.patchValue(this.initialDisputeValues);
+
+      // set violation date and time
+      let violationDate = new Date(response.issuedDate);
+      this.form.get('violationTicket').get('issuedDate').setValue(response.issuedDate);
+      this.form.get('violationTicket').get('violationDate').setValue(this.datePipe.transform(violationDate, "yyyy-MM-dd"));
+      this.form.get('violationTicket').get('violationTime').setValue(this.datePipe.transform(violationDate, "hhmm"));
```

Comment:
Isn't it "HH:mm"? (for 24 hrs)

Reply:
nope the masking in the html takes care of the ':'

Reply:
ok